Makefile

```diff
@@ -72,19 +77,26 @@ clean:
 	rm -rf $(BUILD_DIR)
 
 
+.PHONY: clean-all
+clean-all: clean
+	rm -rf $(VENDOR_DIR)
+	rm -f $(BUILD_IMAGE_MARKER)
+	docker rmi $(IMAGE_REPO):$(IMAGE_TAG)
```

Comment:
Please add force to delete or it will fail most of the time:
`docker` rmi -f $(IMAGE_REPO):$(IMAGE_TAG)